Automattic/jetpack
# Super Cache: Update Boost install/activate card

Fixes https://github.com/Automattic/boost-cloud/issues/237

## Proposed changes:

* Update Boost install/activate card:
  * Spacing between elements;
  * Boost logo;
  * The image on the right side of the card;
  * The alt text of the image.

### Other information:


## Jetpack product discussion



pbNhbs-63V-p2#comment-14458

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup this version of Super Cache;
* Deactivate Jetpack Boost if you have it set up;
* Go to the Super Cache settings page and you should see the updated Boost card.

Before:

![CleanShot 2023-06-12 at 12 26 57@2x](https://github.com/Automattic/jetpack/assets/11799079/aa52121c-ba2e-4e0d-bf8c-a13f3eac7e4d)

After:

![CleanShot 2023-06-12 at 12 25 07@2x](https://github.com/Automattic/jetpack/assets/11799079/01e83fa5-5b1f-4cff-9269-e4a6fb6a1094)
